packages/perseus/src/widgets/radio/base-radio.jsx

```diff
@@ -243,13 +243,25 @@ function BaseRadio(props: Props): React.Node {
     const shouldShowInstructions = !sat;
 
     const responsiveClassName = css(styles.responsiveFieldset);
+    // Note(TB): Each set of questions needs its own unique name to be
+    // considered a radio group; this is what allows only one choice at
+    // a time to be selected for single select questions. Tried
+    // useUniqueIdWithMock from WB, but encountered errors we could not fix.
```

Comment:
Since you ended up not using `useUniqueIdWithMock`, I think you can remove a lot of code from the tests such as `RenderStateRoot` or whatever and all the Jest timer business.